crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -609,19 +612,69 @@ pub enum EventItemOrigin {
     Pagination,
 }
 
+/// What's the status of a reaction?
+#[derive(Clone, Debug)]
+pub enum ReactionStatus {
+    /// It's a local reaction to a remote event.
+    ///
+    /// The handle is missing only in testing contexts.
+    LocalToRemote(Option<SendHandle>),
+    /// It's a remote reaction to a remote event.
+    Remote(OwnedEventId),
```

Comment:
I would rename it `RemoteToRemote` to get the symmetry with `LocalToRemote`, but it's a very personal taste.